build/enso-formatter/src/lib.rs

```diff
@@ -417,16 +416,16 @@ pub fn discover_paths_internal(
     // All files to be formatted should be reachable from the repository root without following
     // any symlinks.
     let md = fs::symlink_metadata(path)?;
-    if md.is_dir() && !path.file_name().contains(&"target") {
+    if md.is_dir() && !path.file_name().is_some_and(|n| n == "target") {
```

Comment:
I really hate all these changes. What lint causes it? Maybe we should disable this lint ...

Reply:
It's not lint, but API change. See https://github.com/enso-org/enso/pull/7109#discussion_r1238439069

I don't think they're _so_ bad, however. They are a bit more verbose, which may be a good thing for reading code.